tests/test_scalar_bars.py

```diff
@@ -85,3 +85,15 @@ def test_add_remove_bar(sphere):
     pl.add_scalar_bar()
     pl.remove_scalar_bar()
     assert len(pl._scalar_bar_slots) == init_slots
+
+
+def test_too_many_scalar_bars(sphere):
+    pl = pv.Plotter()
+
+    with pytest.raises(RuntimeError, match='Maximum number of color'):
+        for i in range(100):
+            mesh = pv.Sphere()
+            mesh[f'{i}'] = range(mesh.n_points)
```

Comment:
```suggestion
            mesh[str(i)] = range(mesh.n_points)
```

Reply:
I meant to submit this with some other comments, but I can only reflect on a recent change by posting this along with that comment, so here goes. We can wait to commit these kinds of changes together later. I'm progressing at snail's pace.